src/PiggyCustomEnchants/EventListener.php

```diff
@@ -350,12 +351,12 @@ public function onSneak(PlayerToggleSneakEvent $event)
     }
 
     /**
-     * @param ProjectileHitBlockEvent $event
+     * @param ProjectileLaunchEvent $event
```

Comment:
Why? We want this to run when a projectile hits a block and not when it is launched. This is for the grappling enchant.

Reply:
maybe a PMMP bug, see the description: ProjectileHitBlockEvent does not have a valid handler list

Reply:
and ProjectHitEvent is an abstract class 0.0